model/project.go

```diff
@@ -372,6 +375,44 @@ type ProjectTask struct {
 	PatchOnly *bool `yaml:"patch_only,omitempty" bson:"patch_only,omitempty"`
 }
 
+type LoggerConfig struct {
+	Type         string `yaml:"type" bson:"type"`
+	SplunkServer string `yaml:"splunk_server,omitempty" bson:"splunk_server,omitempty"`
+	SplunkToken  string `yaml:"splunk_token,omitempty" bson:"splunk_token,omitempty"`
+}
+
+func (c *LoggerConfig) IsValid() error {
+	catcher := grip.NewBasicCatcher()
+	if !util.StringSliceContains(ValidLogSenders, c.Type) {
+		catcher.Add(errors.Errorf("%s is not a valid log sender", c.Type))
+	}
+	if c.Type == string(SplunkLogSender) && c.SplunkServer == "" {
```

Comment:
Are you sure you need to cast to string? https://play.golang.org/p/GgaHMDesizs

Reply:
fixed